ts/modules/draggable-points.src.ts

```diff
@@ -2726,6 +2726,8 @@ function mouseOver(point: Highcharts.Point): void {
     var series = point.series,
         chart = series && series.chart,
         dragDropData = chart && chart.dragDropData;
+    const is3DEnabled = chart && chart.options && chart.options.chart &&
```

Comment:
Instead of checking the options structure, use the test designed for this: `is3d = chart && chart.is3d && chart.is3d()`.

Reply:
And while we're at it, we should consolidate the code to either use only 'const' or only 'var' in the same function block, for the sake of consistency.